cirq/linalg/operator_spaces_test.py

```diff
@@ -232,3 +232,64 @@ def test_expand_is_inverse_of_reconstruct(m1, basis):
     c2 = cirq.expand_matrix_in_orthogonal_basis(m2, basis)
     assert np.allclose(m1, m2)
     assert c1 == c2
+
+
+@pytest.mark.parametrize(
+    'coefficients,exponent',
+    itertools.product((
+        (0, 0, 0, 0),
+        (-1, 0, 0, 0),
+        (0.5, 0, 0, 0),
+        (0.5j, 0, 0, 0),
+        (1, 0, 0, 0),
+        (2, 0, 0, 0),
+        (0, -1, 0, 0),
+        (0, 0.5, 0, 0),
+        (0, 0.5j, 0, 0),
+        (0, 1, 0, 0),
+        (0, 2, 0, 0),
+        (0, 0, -1, 0),
+        (0, 0, 0.5, 0),
+        (0, 0, 0.5j, 0),
+        (0, 0, 1, 0),
+        (0, 0, 2, 0),
+        (0, 0, 0, -1),
+        (0, 0, 0, 0.5),
+        (0, 0, 0, 0.5j),
+        (0, 0, 0, 1),
+        (0, 0, 0, 2),
+        (0, -1, 0, -1),
+        (0, 1, 0, 1j),
+        (0, 0.5, 0, 0.5),
+        (0, 0.5j, 0, 0.5j),
+        (0, 0.5, 0, 0.5j),
+        (0, 1, 0, 1),
+        (0, 2, 0, 2),
+        (0, 0.5, 0.5, 0.5),
+        (0, 1, 1, 1),
+        (0, 1.1j, 0.5 - 0.4j, 0.9),
+        (0.7j, 1.1j, 0.5 - 0.4j, 0.9),
+        (0.25, 0.25, 0.25, 0.25),
+        (0.25j, 0.25j, 0.25j, 0.25j),
+        (0.4, 0, 0.5, 0),
+        (1, 2, 3, 4),
+        (-1, -2, -3, -4),
+        (-1, -2, 3, 4),
+        (1j, 2j, 3j, 4j),
+        (1j, 2j, 3, 4),
+    ), (0, 1, 2, 3, 4, 5, 10, 11, 20, 21, 100, 101)),
```

Comment:
maybe get rid of 10, 11, 20, 21?  don't quite see how those would pass when 100, 101 would succeed.

Reply:
Done.